ddtrace/compat.py

```diff
@@ -1,8 +1,16 @@
 import sys
+import platform
 
 
 PY2 = sys.version_info[0] == 2
 
+# Default tags we use when we initialize the tracer
+DEFAULT_TAGS = {
```

Comment:
Oh even if it's more handy that way, it's better to put here immutable strings (so `PYTHON_VERSION=` and `PYTHON_INTERPRETER=`), and keep the dictionary in `tracer.py`. This is still a `compat.py` module that is used to keep values for compatibility reason and not to keep shared objects between modules. Sorry for not being so clear in the previous comment!